lilac2/packages.py

```diff
@@ -42,8 +61,21 @@ def get_dependency_map(
       dependers = rmap[pkgbase]
       for dd in dependers:
         map[dd].update(deps)
+    if pkgbase in norm_rmap:
+      deps = norm_map[pkgbase]
+      dependers = norm_rmap[pkgbase]
+      for dd in dependers:
+        norm_map[dd].update(deps)
+
+  build_dep_map: DefaultDict[str, Set[Dependency]] = defaultdict(set)
+  for pkgbase, info in lilacinfos.items():
+    build_deps = build_dep_map[pkgbase]
+    build_deps.update(norm_map[pkgbase])
+    for d in info.repo_makedepends:
+      build_deps.add(depman.get(d))
+      build_deps.update(norm_map[d])
```

Comment:
`norm_map` is indexed by pkgbase, but `d` is `(pkgbase, pkgname)`.
```suggestion
      build_deps.update(norm_map[d[0]])
```

Reply:
I changed this to use `Dependency.pkgdir.name` to be consistent with the other code. `DependencyManager` supports using either `pkgbase` or `(pkgbase, pkgname)`, and the test uses only `pkgbase`.